From 7a1fa871819bfcd61d7b21b0e3678bbfdd896227 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:50:51 -0400
Subject: [tools] perf/symbols: Remove needless static binary_type array

Message-id: <1408953210-26343-164-git-send-email-jolsa@redhat.com>
Patchwork-id: 88847
O-Subject: [PATCH RHEL7.1 BZ1133083 163/322] perf symbols: Remove needless static binary_type array
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 631d34b5626a8de828f3ab8da54013293097e678
Author: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Mon Dec 16 16:57:43 2013 -0300

description
===========
There are no references to that array anywhere, it is only used to try
a series of "binary" types in turn, always setting dso->data_type till
one can be used.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/dso.c b/tools/perf/util/dso.c
index 436922f..e7f4449 100644
--- a/tools/perf/util/dso.c
+++ b/tools/perf/util/dso.c
@@ -161,7 +161,7 @@ static int open_dso(struct dso *dso, struct machine *machine)
 
 int dso__data_fd(struct dso *dso, struct machine *machine)
 {
- static enum dso_binary_type binary_type_data[] = {
+ enum dso_binary_type binary_type_data[] = {
   DSO_BINARY_TYPE__BUILD_ID_CACHE,
   DSO_BINARY_TYPE__SYSTEM_PATH_DSO,
   DSO_BINARY_TYPE__NOT_FOUND,
-- 
1.7.1